src/HostAgnostic.props

```diff
@@ -2,6 +2,7 @@
 <Project>
   <PropertyGroup>
     <BannedSymbolsOptOut Condition="$(IsTestProject)">true</BannedSymbolsOptOut>
+    <NewtonsoftJsonVersion>9.0.1</NewtonsoftJsonVersion>
```

Comment:
As of 16.2, VS ships with 12.0.2 and we're supposed to update "at your convenience", so can we just set this to 12.0.2 and not need the special case for integration tests?

Reply:
Ah, good to know, I'll do a followup change. I need to make sure we don't have a handshake with other components here.

Reply:
https://github.com/dotnet/project-system/pull/5079